tests/unit_tests/bittensor_tests/utils/test_fastsync.py

```diff
@@ -0,0 +1,48 @@
+import unittest
+from unittest.mock import patch
+import pytest
+import bittensor
+try:
+    from subtensorapi.exceptions import *
+    fastsync_imported = True
+except ImportError:
+    fastsync_imported = False
```

Comment:
why do we have this exception case?

Reply:
wouldn't be better to isolate the code from this library???

Reply:
This is for the exceptions only. I wanted to test the handling of exceptions caused during the interface with the subtensor API library

Reply:
I have more tests with the class itself in the library

Reply:
So we can avoid this here right??? What's the scenario where the `ImportError` is raised?

Reply:
if the whole test suite is run on a machine without the package installed

Reply:
i would say that if the test suite is running without this package installed will fail. We do not need this and It does not seems like a good practice to me.

What do you think about this?

Reply:
Just let the tests fail? I am primarily concerned with development and local testing happening on a machine that doesn't support `subtensorapi`
Although, we don't officially support any OS that doesn't support `subtensorapi`, so maybe this isn't a valid concern.